x/pool-incentives/keeper/keeper.go

```diff
@@ -27,12 +27,9 @@ type Keeper struct {
 	bankKeeper       types.BankKeeper
 	incentivesKeeper types.IncentivesKeeper
 	distrKeeper      types.DistrKeeper
-
-	communityPoolName string // name of the Community pool ModuleAccount (Maybe the distribution module)
-	feeCollectorName  string // name of the FeeCollector ModuleAccount
 }
 
-func NewKeeper(cdc codec.BinaryCodec, storeKey sdk.StoreKey, paramSpace paramtypes.Subspace, accountKeeper types.AccountKeeper, bankKeeper types.BankKeeper, incentivesKeeper types.IncentivesKeeper, distrKeeper types.DistrKeeper, communityPoolName string, feeCollectorName string) Keeper {
+func NewKeeper(cdc codec.BinaryCodec, storeKey sdk.StoreKey, paramSpace paramtypes.Subspace, accountKeeper types.AccountKeeper, bankKeeper types.BankKeeper, incentivesKeeper types.IncentivesKeeper, distrKeeper types.DistrKeeper) Keeper {
```

Comment:
Does it make sense to rename `DistKeeper` interface to `CommunityPoolKeeper` here as well?

Reply:
It does